Automattic/wp-calypso
# VideoPress onboarding v2 : buttons feedback

#### Proposed Changes

This PR adds button feedback for site options and plan selection steps.

#### Testing Instructions

* Apply this PR
* Navigate to `http://calypso.localhost:3000/setup/intro?flow=videopress`
* Start the flow until site options
* ✅ The submit button should be disabled
* Add spaces in "site title"
* ✅ The submit button should still be disabled
* Set a site title
* ✅ The submit button should be enabled
* Go to the next step, choose a domain later
* Select a plan
* ✅ You should see a button animation while the site is being created

Fix Automattic/greenhouse#1345
